### PR TITLE
Add external links to the Event page

### PR DESCRIPTION
**Pull request by torsava**
Fixes issue #38

### CODE REVIEW

**Comment by torsava** [comment]
I chose the formulations:

    External link for this event
    External links for this event:  [ 1 ]  [ 2 ] ...

However, if you can think of a better ones, I'm open to suggestions.

**Comment by torsava** [comment]
**Changes:**
- Changed the position to the very bottom of the page.
- Updated the English text to a better sounding one.
- Added a Czech translation.
- Entries are now shown using their base URL, not a number.

File: pyvocz/templates/event.html [hunk 1]
```diff
@@ -125,6 +125,20 @@
             <a href="http://www.google.com/maps/place/{{ event.venue.latitude }},{{ event.venue.longitude }} ">Google Maps</a>.
         </div>
     {% endif %}
+
+    {% if event.links %}
+        <div>
+            {{ tr('Jinde na webu:', 'Elsewhere on the web:') }}
+            {% for link in event.links|sort(attribute='index') %}
+                <a href="{{ link.url }}">{{ urlparse(link.url).netloc }}</a>
+                {%- if loop.index < event.links|length -%}
```
Comment (encukou) [inline pyvocz/templates/event.html]: You can use `loop.last`, see http://jinja.pocoo.org/docs/2.9/templates/#for

Comment (torsava) [inline pyvocz/templates/event.html]: Done!

**Comment by torsava** [comment]
**Changes:**
- Used loop.last for simplicity
- Moved the links below the description for better visibility and not to be lumped together with the location
- Removed 'www.' from the links if present.

**Comment by encukou** [comment]
Looks great, thanks!

**Pull request merged**